app.rb

```diff
@@ -47,6 +46,21 @@ def log_comparison(comparison)
     puts line.to_json
   end
 
+  def log_level(comparison)
+    if (comparison[:different_keys].empty? || comparison[:different_keys] == "N/A") &&
+        matches?(comparison, :status) &&
+        matches?(comparison, :body_size)
+      :info
+    else
+      :warn
+    end
+  end
+
+  def matches?(comparison, field)
+    comparison.fetch(:stats, {}).fetch(:primary_response, {}).fetch(field, nil) ==
+      comparison.fetch(:stats, {}).fetch(:secondary_response, {}).fetch(field, nil)
```

Comment:
[Can you dig it?](https://www.youtube.com/watch?v=sGSHIidlesQ)

```suggestion
    comparison.dig(:stats, :primary_response, field, nil) == comparison.dig(:stats, :secondary_response, field, nil)
```

Reply:
Much cleaner :+1: